src/aat/resources/features/F-201 - New Access Control Get Case/Grant_Role_Assignment.td.json

```diff
@@ -0,0 +1,62 @@
+{
+  "_guid_" : "GRANT_CASE_ROLE_ASSIGNMENT",
+
+  "productName": "Role Assignment Service",
+  "operationName": "post role assignment",
+  "method": "POST",
+  "uri": "http://localhost:4096/am/role-assignments",
```

Comment:
we need to make this host dynamic or env var  as this wont work i higher up envs. we can use ROLE_ASSIGNMENT_HOST env var